src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -574,21 +575,41 @@ def handleSpecialCERNMergeSettings(self, funcName):
         Enable lazy-download for fastCloning for all CMSSW_7_5 jobs (currently off)
         Enable lazy-download for all merge jobs
         """
-        cmsswVersion = os.environ['CMSSW_VERSION']
+        cmsswVersion = os.environ.get('CMSSW_VERSION', "")
         if cmsswVersion.startswith("CMSSW_7_5") and False:
             print "Using fastCloning/lazydownload"
-            self.process.add_(
-                cms.Service("SiteLocalConfigService",
-                            overrideSourceCloneCacheHintDir=cms.untracked.string("lazy-download")
-                            )
-                )
+            self.process.add_(cms.Service("SiteLocalConfigService",
+                                          overrideSourceCloneCacheHintDir = cms.untracked.string("lazy-download")))
         elif funcName == "merge":
             print "Using lazydownload"
-            self.process.add_(
-                cms.Service("SiteLocalConfigService",
-                            overrideSourceCacheHintDir=cms.untracked.string("lazy-download")
-                            )
-                )
+            self.process.add_(cms.Service("SiteLocalConfigService",
+                                          overrideSourceCacheHintDir = cms.untracked.string("lazy-download")))
+        return
+
+    def handleCondorStatusService(self):
+        """
+        _handleCondorStatusService_
+
+        Enable CondorStatusService for CMSSW releases that support it.
+
```

Comment:
There is no document that lists what is supported, I just browsed the CMSSW code repository and checked the versions that have the CondorStatusService service plugin.